core/src/main/java/bisq/core/btc/wallet/TradeWalletService.java

```diff
@@ -1193,10 +1196,6 @@ private RawTransactionInput getRawInputFromTransactionInput(@NotNull Transaction
 
     private byte[] getMakersScriptSigProgram(TransactionInput transactionInput) throws TransactionVerificationException {
         byte[] scriptProgram = transactionInput.getScriptSig().getProgram();
-        if (scriptProgram.length == 0) {
-            throw new TransactionVerificationException("Inputs from maker not signed.");
-        }
-
         return scriptProgram;
```

Comment:
Variable is redundant now. Also does not throw exception anymore

Reply:
This all started with scriptProgram.length == 0 being valid for a segwit input.
I just removed the method and moved its content to where it was being used, i.e. takerSignsDepositTx().
See takerSignsDepositTx():
```
                byte[] makersScriptSigProgram = makersInput.getScriptSig().getProgram();
                if (makersScriptSigProgram.length == 0 && TransactionWitness.EMPTY.equals(makersInput.getWitness())) {
                    throw new TransactionVerificationException("Inputs from maker not signed.");
                }
```
See https://github.com/bisq-network/bisq/pull/4612/commits/7eb808d2bf0af0b4b292ff44289459eaabcbcde4

Reply:
Yes, I saw. But now the method does not throw TransactionVerificationException anymore.

Reply:
I saw you removed the method. Good so...